scripts/bin_to_c.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# SPDX-License-Identifier: BSD-2-Clause
+#
+# Copyright (c) 2018, Linaro Limited
+#
+
+import argparse
+import array
+import os
+import re
+
+def get_args():
+
+	parser = argparse.ArgumentParser(description='Converts a binary '
+		'file into C source file defining binary data as a'
+		'constant byte array.')
+
+	parser.add_argument('--dtb', required=True,
+		help='Path to the input binary file')
+
+	parser.add_argument('--label', required=True,
+		help='Label for the generated table in the C source file.')
+
+	parser.add_argument('--out', required=True,
+		help='Path of the output C file')
+
+	return parser.parse_args()
+
+def main():
+
+	args = get_args();
+
+	with open(args.dtb, 'rb') as indata:
+		bytes = indata.read()
+		size = len(bytes)
+
+	f = open(args.out, 'w')
+	f.write('/* Generated from ' + args.dtb + ' by ' +
+		os.path.basename(__file__) + ' */\n\n')
```

Comment:
Do we really need to let people know about the full path (which is machine/developer unique). Isn't just the name of the script sufficient? When I was running the script I get:
```c
/* Generated from /home/jbech/devel/optee_projects/qemu/linux/arch/arm/boot/dts/vexpress-v2p-ca9.dtb by bin_to_c.py */
```

Reply:
If we want to uniquely be able identify it, then it's better to have the sha1 or so for the DTB shown as a comment. I.e.
```
Generated from foobar.dtb (sha1: 988881adc9fc3655077dc2d4d757d480b5ea0e11) by bin_to_c.py
```

Reply:
The output is a "private" file, it stays in the build environment. The comment won't be included in any way in the TEE binary.
Main purpose is for troubleshooting build/dependencies issues, things like that. So we definitely don't want the file name only, there has to be some path (relative or absolute it doesn't matter, btw the path you are seeing is absolute not because of the Python code but because the script was called with an absolute path I think).
Having a SHA1 could be useful but a bit overkill, usually checking file dates is enough.
In any case there is no reason this line should be different than in `ta_bin_to_c.py`.

Reply:
Ok, if the file is never moved or included in other builds (i.e, saved as pre-generated), then I'm fine with keeping it as it is.